app/assets/javascripts/govuk_publishing_components/analytics/page-content.js

```diff
@@ -45,7 +45,7 @@
       case isDocumentCollectionPage():
         return document.querySelectorAll('.document-collection .group-document-list li a').length
       case isMainstreamBrowsePage():
-        return countVisible(document.querySelectorAll('#subsection ul a')) || document.querySelectorAll('#section ul a').length || document.querySelectorAll('#root ul a').length
+        return countVisible(document.querySelectorAll('#subsection ul a')) || document.querySelectorAll('#section ul a').length || document.querySelectorAll('#root ul a').length || document.querySelectorAll('.gem-c-cards__link').length
```

Comment:
Generally we try to avoid referencing a class name internal to a component outside of a component, because components are subject to change and doing so can be brittle - but that might not always be possible. Is there an alternative way to count items on the new browse page without referencing a component class?

Reply:
>so can be brittle

100% agreed, how does a `data-` approach sound instead for this? Adding an attribute to the component - for tracking.  I feel like doing DOM selections element is equally quite brittle.  I mean it's less likely the `ul > a` format will change however it might (eg an `ol` for curated content)

Reply:
Yeah, a data attribute approach could work, because those would be defined on the browse page and passed to the component, so much more specific and controllable.